DDCore/include/DD4hep/CylindricalGridPhiZ.h

```diff
@@ -0,0 +1,111 @@
+//==========================================================================
+//  AIDA Detector description implementation 
+//--------------------------------------------------------------------------
+// Copyright (C) Organisation europeenne pour la Recherche nucleaire (CERN)
+// All rights reserved.
+//
+// For the licensing terms see $DD4hepINSTALL/LICENSE.
+// For the list of contributors see $DD4hepINSTALL/doc/CREDITS.
+//
+//  \author   Markus Frank
+//  \date     2016-10-18
+//  \version  1.0
+//
+//==========================================================================
+#ifndef DD4HEP_CYLINDRICALGRIDPHIZ_H
+#define DD4HEP_CYLINDRICALGRIDPHIZ_H 1
+
+// Framework include files
+#include <DD4hep/Segmentations.h>
+
+/// Namespace for the AIDA detector description toolkit
+namespace dd4hep {
+
+  /// Namespace for base segmentations
+  namespace DDSegmentation  {    class CylindricalGridPhiZ;  }
+    
+  /// We need some abbreviation to make the code more readable.
+  typedef Handle<SegmentationWrapper<DDSegmentation::CylindricalGridPhiZ> > CylindricalGridPhiZHandle;
+ 
+  /// Implementation class for the grid PhiZ segmentation.
+  /**
+   *  Concrete user handle to serve specific needs of client code
+   *  which requires access to the base functionality not served
+   *  by the super-class Segmentation.
+   *
+   *  Note:
+   *  We only check the validity of the underlying handle.
+   *  If for whatever reason the implementation object is not valid
+   *  This is not checked.
+   *  In principle this CANNOT happen unless some brain-dead has
+   *  fiddled with the handled object directly.....
+   *
+   *  Note:
+   *  The handle base corrsponding to this object in for
```

Comment:
```suggestion
   *  The handle base corresponding to this object in for
```

Reply:
I committed your suggested change (or so I think: I'm not familiar with this answer-to-reviewer procedure) . Turns out that "grep -R corrsponding" returns 14 matches in "./DD4hep/DDCore".

Reply:
I have added a purely cosmetics commit, w/ all of your suggestions concerning comments.